arangod/RocksDBEngine/RocksDBEngine.h

```diff
@@ -448,6 +448,22 @@ class RocksDBEngine final : public StorageEngine {
   }
 #endif
 
+  class RocksDBSnapshot final : public StorageSnapshot {
+   public:
+    explicit RocksDBSnapshot(rocksdb::DB& db) noexcept : _snapshot(&db) {}
+
+    TRI_voc_tick_t tick() const noexcept override {
+      return _snapshot.snapshot()->GetSequenceNumber();
+    }
+
+    decltype(auto) getSnapshot() const { return _snapshot.snapshot(); }
+
+   private:
+    mutable rocksdb::ManagedSnapshot _snapshot;
```

Comment:
As we are using `rocksdb::ManagedSnapshot` here now, we should add an explicit include at the top of the file to include `<rocksdb/snapshot.h>`.